addons/common/functions/fnc_fireVLS.sqf

```diff
@@ -0,0 +1,65 @@
+#include "script_component.hpp"
+/*
+ * Author: Kex
+ * Orders a VLS unit to commence fire burst on the given position (silently).
+ *
+ * Arguments:
+ * 0: VLS unit <OBJECT>
+ * 1: Target position <ARRAY>
+ * 2: Spread <NUMBER>
+ * 3: Magazine class <STRING>
+ * 4: Number of rounds <NUMBER>
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * [_vlsUnit, getPos player, _magazine, 1] call zen_common_fnc_fireVLS;
+ *
+ * Public: No
+ */
+
+#define GUNNER_TURRET [0]
+
+params [["_unit", objNull, [objNull]], ["_targetPosition", [0, 0, 0], [[]], 3], ["_spread", 0, [0]], ["_magazine", "", [""]], ["_rounds", 1, [0]]];
+
+private _muzzle = (_unit weaponsTurret GUNNER_TURRET) param [0, ""];
+
+private _eta = [_unit, _targetPosition, _magazine] call FUNC(getArtilleryETA);
+private _reloadTime = [_unit, _muzzle, GUNNER_TURRET] call FUNC(getWeaponReloadTime);
+
+// Load magazine even if it is the right one in order to suppress a possible ongoing magazine loading
+[_unit, GUNNER_TURRET, _muzzle, _magazine] call FUNC(loadMagazineInstant);
+
+_unit setVariable [QGVAR(roundCounter), 0];
+[
+    {
+        params ["_args", "_handle"];
+        _args params ["_unit", "_targetPosition", "_spread", "_muzzle", "_eta", "_reloadTime", "_rounds"];
+
+        // VLS needs an actual dummy target to fire at
+        _targetPosition = [_targetPosition, _spread] call CBA_fnc_randPos;
+        private _logicGroup = createGroup sideLogic;
+        private _target = _logicGroup createUnit ["Module_F", _targetPosition, [], 0, "CAN_COLLIDE"];
+        _logicGroup deleteGroupWhenEmpty true;
```

Comment:
Can use alternate syntax for `createGroup`. -> `createGroup [side, deleteWhenEmpty]`